adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -810,6 +811,8 @@ private IWindowComponent wrapActivity(final Activity activity) {
         }
 
         return new IWindowComponent() {
+            //CHECKSTYLE:OFF: checkstyle:MemberName
+            //CHECKSTYLE:OFF: checkstyle:VisibilityModifier
             Activity mRefActivity = activity;
```

Comment:
sorry about this, but how about just making this private? does it break anything?